### PR TITLE
Prometheus saves data on local fs

### PR DESCRIPTION
**Pull request by avimehenwal**
Fixes 

```
prometheus_1  | level=error ts=2019-11-04T21:57:29.359Z caller=main.go:736 err="error loading config from \"/etc/prometheus/prometheus.yml\": couldn't load configuration (--config.file=\"/etc/prometheus/prometheus.yml\"): parsing YAML file /etc/prometheus/prometheus.yml: openstack SD configuration requires a region"
```

and

> unable to write TS data files to local filesystem 

### Tests

1. Tested on localmachine :ballot_box_with_check: 
2. on bf2 :ballot_box_with_check: 
 
**NOTE:** did not do the full run

### CODE REVIEW

File: prometheus/prometheus.yml [hunk 1]
```diff
@@ -1,74 +1,22 @@
-#  Copyright (c) 2019 SONATA-NFV, 5GTANGO, Paderborn University
```
Comment (qarawlus) [inline prometheus/prometheus.yml]: Please restore the copyright notice

File: prometheus/prometheus.yml [hunk 2]
```diff
@@ -1,74 +1,22 @@
...
-  - job_name: 'custom-vnf0'
-    static_configs:
-      - targets: ['fgcn-tango-5.cs.upb.de:9110']
-  #- job_name: 'custom-vnf1'
-  #  static_configs:
-  #    - targets: ['fgcn-tango-5.cs.upb.de:9111']
-  #- job_name: 'custom-vnf2'
-  #  static_configs:
-  #    - targets: ['fgcn-tango-5.cs.upb.de:9112']
+      - targets: ['fgcn-backflip1.cs.uni-paderborn.de:9100']
```
Comment (qarawlus) [inline prometheus/prometheus.yml:41]: Why backflip1?

**Review by qarawlus — CHANGES_REQUESTED** [review]
Please check my inline comments. Also, please test on a VM if you are not sure that the changes you made locally will work on the VM

File: prometheus/prometheus.yml (hunk 1 above)
Comment (avimehenwal) [reply prometheus/prometheus.yml]: done

File: prometheus/prometheus.yml (hunk 2 above)
Comment (avimehenwal) [reply prometheus/prometheus.yml:41]: **POC** that prometheus can collect data on nodes with running `node_exporter`. Its a sample configuration to test static nodes.

**Comment by avimehenwal** [comment]
Screenshot_20191104_232719

**Pull request merged**